examples-gallery/plot_parallel_park.py

```diff
@@ -185,10 +186,14 @@
 
 # %%
 # Find the optimal solution.
+initial_guess = np.load('parallel_park_solution.npy')
```

Comment:
If we add these to examples, we need to explain to the reader why we are doing so.

Reply:
> If we add these to examples, we need to explain to the reader why we are doing so.

I will correct. Just slipped.